filament/backend/include/private/backend/CommandStream.h

```diff
@@ -153,10 +153,10 @@ struct CommandType<void (Driver::*)(ARGS...)> {
         static inline void execute(M&& method, D&& driver, CommandBase* base, intptr_t* next) noexcept {
             Command* self = static_cast<Command*>(base);
             *next = align(sizeof(Command));
-            if (DEBUG_COMMAND_STREAM) {
+#if DEBUG_COMMAND_STREAM
```

Comment:
Why this change here?

Reply:
Because when you do:

```
if (false) 
    self->log();
```

the MSVC compiler, when building in debug mode, generates a call to self->log() in the executable and we get an undefined symbol, because the function is defined in CommandStream.cpp with:


```
#if DEBUG_COMMAND_STREAM
#define DECL_DRIVER_API_SYNCHRONOUS(RetType, methodName, paramsDecl, params)
#define DECL_DRIVER_API(methodName, paramsDecl, params) \
    template void CommandType<decltype(&Driver::methodName)>::Command<&Driver::methodName>::log() noexcept;
#define DECL_DRIVER_API_RETURN(RetType, methodName, paramsDecl, params) \
    template void CommandType<decltype(&Driver::methodName##R)>::Command<&Driver::methodName##R>::log() noexcept;
#include "private/backend/DriverAPI.inc"
#endif
```

The change just uses the preprocessor (#if) in both `CommandStream.h` and `CommandStream.cpp` so it is consistent, instead if using if in the .h and #if ion the .cpp.